src/classes/CRLP_RollupContact_TEST.cls

```diff
@@ -348,6 +348,9 @@ private class CRLP_RollupContact_TEST {
                 npe01__Do_Not_Automatically_Create_Payment__c = true,
                 Type = 'New'
         ));
+        if (tt == TestType.TestWithAlwaysRollupToPrimary) {
+            donationYears.add(orgDonationDate.year().format());
```

Comment:
This will need to use the same `UTIL_String.removeNonNumericCharacters()` method as used on line 318.

Reply:
Why can't I use the actual CloseDate value? I thought this was what you were suggesting in chat as a replacement for removeNonNumeric ... ?

Reply:
`orgDonationDate.year().format()` will return `2,019`.

Reply:
Ah, okay. I wonder how the test was passing with that in there!

Reply:
Line 318 uses the method to strip out the comma from the year before adding it to the `donationYears` set. Line 352 has to add the year in the same format.